content/blog/2018/incremental-traffic-management/index.md

```diff
@@ -0,0 +1,225 @@
+
+First, **setup and configure [GKE](https://istio.io/docs/setup/kubernetes/platform-setup/gke/)**:
+
+{{< text bash >}}
+$ gcloud container clusters create istio-inc --zone us-central1-f
+$ gcloud container clusters get-credentials istio-inc
+$ kubectl create clusterrolebinding cluster-admin-binding \
+   --clusterrole=cluster-admin \
+   --user=$(gcloud config get-value core/account)
+{{< /text >}}
+
+Next, **[install Helm](https://docs.helm.sh/using_helm/#installing-helm) and [generate a minimal Istio install](https://istio.io/docs/setup/kubernetes/minimal-install/)** --  only traffic management components:
+
+{{< text bash >}}
+$ helm template install/kubernetes/helm/istio \
+  --name istio \
+  --namespace istio-system \
+  --set security.enabled=false \
+  --set galley.enabled=false \
+  --set sidecarInjectorWebhook.enabled=false \
+  --set mixer.enabled=false \
+  --set prometheus.enabled=false \
+  --set global.proxy.envoyStatsd.enabled=false \
+  --set pilot.sidecar=false > istio-minimal.yaml
+{{< /text >}}
+
+Then **create the istio-system namespace and deploy Istio**:
+
+{{< text bash >}}
+$ kubectl create namespace istio-system
+$ kubectl apply -f istio-minimal.yaml
+{{< /text >}}
+
+Next, **deploy the Bookinfo sample** without the Istio sidecar containers:
+
+{{< text bash >}}
+$ kubectl apply -f samples/bookinfo/platform/kube/bookinfo.yaml
+{{< /text >}}
+
+Now, **configure a new Gateway** that allows access to the reviews service from outside the Istio mesh, a new VirtualService that splits traffic evenly between v1 and v2 of the reviews service, and a set of new DestinationRules that match destination subsets to service versions:
+
+{{< text bash >}}
+$ cat <<EOF | kubectl apply -f -
+apiVersion: networking.istio.io/v1alpha3
+kind: Gateway
+metadata:
+  name: reviews-gateway
+spec:
+  selector:
+    istio: ingressgateway # use istio default controller
+  servers:
+  - port:
+      number: 80
+      name: http
+      protocol: HTTP
+    hosts:
+    - "*"
+---
+apiVersion: networking.istio.io/v1alpha3
+kind: VirtualService
+metadata:
+  name: reviews
+spec:
+  hosts:
+  - "*"
+  gateways:
+  - reviews-gateway
+  http:
+  - match:
+    - uri:
+        prefix: /reviews
+    route:
+    - destination:
+        host: reviews
+        subset: v1
+      weight: 50
+    - destination:
+        host: reviews
+        subset: v2
+      weight: 50
+---
+apiVersion: networking.istio.io/v1alpha3
+kind: DestinationRule
+metadata:
+  name: reviews
+spec:
+  host: reviews
+  subsets:
+  - name: v1
+    labels:
+      version: v1
+  - name: v2
+    labels:
+      version: v2
+  - name: v3
+    labels:
+      version: v3
+EOF
+{{< /text >}}
+
+Finally, **deploy a Pod that you can use for testing** with `curl`:
+
+{{< text bash >}}
+$ kubectl apply -f samples/sleep/sleep.yaml
+{{< /text >}}
```

Comment:
You should point out that the automatic sidecar injection is not enabled, so this is just starting the sleep pod with 1 container, instead of the usual 2 (app and sidecar proxy) that people are used to when using it.

Reply:
Addressed by including a comment that the sleep Pod is deployed **without** the sidecar container